### PR TITLE
Aryan/tiptap changes/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/components/tiptapEditor/slashMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+            default:
+                break;
+        }
+    
+        setShowSlashMenu(false);
+    };    
+
+    return(
+        <>
+           <div className="slash-menu position-absolute align-items-center d-flex flex-column bg-white py-2" style={{top: `${slashMenuPosition.top}px`,left: `${slashMenuPosition.left}px`}}>
```
Comment (Saurabh186) [inline src/components/tiptapEditor/slashMenu.jsx:218]: why r ussing bg-white

File: src/components/tiptapEditor/slashMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                <div className="slash-menu-container">
+                    <div className="search-bar d-flex align-items-center p-16  m-2">
```
Comment (Saurabh186) [inline src/components/tiptapEditor/slashMenu.jsx:220]: p-16 ???? what??

File: src/components/tiptapEditor/slashMenu.jsx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                        <span className='search-icon mr-2 ml-2 align-items-center justify-content-between'><BsSearch size={16} /></span>
```
Comment (Saurabh186) [inline src/components/tiptapEditor/slashMenu.jsx:221]: not using size={16} by default 16 he hota hai

File: src/components/tiptapEditor/tiptap.jsx [hunk 4]
```diff
@@ -322,35 +127,43 @@ export default function Tiptap({ provider, ydoc, isInlineEditor, disabled, initi
...
-              });
+            const selection = window.getSelection();
+            const container = document.querySelector('.textEditorContainer');
+            const containerRect = container.getBoundingClientRect();
+            const containerOffsetLeft = containerRect.left;
+            const editorPaddingTop = parseFloat(window.getComputedStyle(container).paddingTop) || 0;
+    
+            if (selection.rangeCount > 0) {
+                const range = selection.getRangeAt(0);
+                const rect = range.getBoundingClientRect();
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.jsx:146]: remove spacing

**Review by Saurabh186 — APPROVED** [review]

File: src/components/tiptapEditor/slashMenu.jsx (hunk 1 above)
Comment (ViaSocket-Git) [reply src/components/tiptapEditor/slashMenu.jsx:218]: Resolved

File: src/components/tiptapEditor/slashMenu.jsx (hunk 2 above)
Comment (ViaSocket-Git) [reply src/components/tiptapEditor/slashMenu.jsx:220]: Resolved

File: src/components/tiptapEditor/tiptap.jsx (hunk 4 above)
Comment (ViaSocket-Git) [reply src/components/tiptapEditor/tiptap.jsx:146]: Resolved

File: src/components/tiptapEditor/slashMenu.jsx (hunk 3 above)
Comment (ViaSocket-Git) [reply src/components/tiptapEditor/slashMenu.jsx:221]: Resolved

**Review by Saurabh186 — APPROVED** [review]

**Pull request merged**